packages/ui/src/common/components/typography/Fraction.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react'
+import styled, { css } from 'styled-components'
+
+import { Colors } from '../../constants'
+
+import { TextInlineMedium } from './Text'
+
+interface FractionProps {
+  num: string | number
+  den: string | number
+  className?: string
+  sameSize?: boolean
+}
+
+export const Fraction = React.memo(({ num, den, className, sameSize }: FractionProps) => {
+  return (
+    <FractionBlock className={className} sameSize={sameSize} lighter value>
+      <FractionNumerator>{num ? num : 0}</FractionNumerator>
```

Comment:
```suggestion
      <FractionNumerator>{num ?? 0}</FractionNumerator>
```

For undefined check, you can use nullish coalescing operator `??` - does the same but nicer syntax :)